requirements.txt

```diff
@@ -74,8 +70,3 @@ sphinx-rtd-theme==0.1.9 \
     --hash=sha256:46c2ca8b482b0398b9621bb5a8ec7c9c5c7226926e5363e18d744f5f0d39d78f \
     --hash=sha256:273846f8aacac32bf9542365a593b495b68d8035c2e382c9ccedcac387c9a0a1
 virtualenv==15.0.1 --hash=sha256:13ce1079910a6bc60e2ce1d79813a99f30b2fd1e571427fcde1fabb0ff4c436c
-waitress==0.9.0 --hash=sha256:57192b7b9f26eae770f5bbf5a3cfeb781011305b1b6ca79a59d769dc267989a2
-WebOb==1.6.0 --hash=sha256:69356a75980172f3e76717f6899b6c75d95ccdc7a2cb6a3b8bd6a7de3fc72401
-WebTest==2.0.23 \
-    --hash=sha256:0831371d415fac39177d1fab4e180510713401a71edd9181373efe123e85cb3e \
-    --hash=sha256:879da5c2310b75c74dffa23a45a7a74d09c2a0fcc7e7a128b4f92a4f64722e97
```

Comment:
We ditched WebTest--I wrote some test infrastructure that does everything I needed it to do. Because of that, we got to nix a bunch of requirements.